apps/src/lab2/views/Loading.tsx

```diff
@@ -23,10 +24,22 @@ const Loading: React.FunctionComponent<LoadingProps> = ({
     ? moduleStyles.fadeLoading
     : moduleStyles.fadeLoaded;
 
+  const background = useAppSelector(
+    state =>
+      state.progress.lessons?.find(l => l.id === state.progress.currentLessonId)
```

Comment:
For later - might be good to create a `getCurrentLesson` selector that does this lookup? I feel like we do something similar in a few different places.

Reply:
Good call.